src/Form.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import FormSection from "./FormSection";
+
+class Form extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			title: "",
+			userFirstName: "",
+			userLastName: "",
+			callerNumber: "",
+		};
+		this.handleChange = this.handleChange.bind(this);
```

Comment:
if you make `handleChange` a arrow function, then I don't think you need the bind

Reply:
That really does seems a lot easier.